<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>781</width>
    <height>607</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>picogen:CosyGui</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#centralWidget {
	background: qlineargradient(x1: 0, y1: 0, x2:0, y2: 1,
		stop: 0 #b6b6b6, stop: 0.08 #8f8f8f,
		stop: 0.4 #727272,
		stop: 0.9 #5c5c5c, stop: 1 #434343);
}

/* Main Frame *************************************************/
#mainFrame {
       background-color: green;
}



/* Navigation Frame *************************************************/
/* largely inspired by http://thesmithfam.org/blog/2009/09/17/qt-stylesheets-button-bar-tutorial/ */
#navigationFrame {
	border: none;
	background: qlineargradient(x1: 0, y1: 0, x2:0, y2: 1,
		stop: 0 #a6a6a6, stop: 0.08 #7f7f7f,
		stop: 0.4 #626262,
		stop: 0.9 #4c4c4c, stop: 1 #333333);
}
#navigationFrame QPushButton {
	color: #333;
	border: 2px solid #555;
	border-radius: 11px;
	padding: 5px;
	background: qradialgradient(cx: 0.3, cy: -0.4,
		fx: 0.3, fy: -0.4,
		radius: 1.35, stop: 0 #fff, stop: 1 #888);
	min-width: 80px;
}
#navigationFrame QPushButton:checked {
	color: #333;
	border: 2px solid #555;
	border-radius: 11px;
	padding: 5px;
	background: qradialgradient(cx: 0.3, cy: -0.4,
		fx: 0.3, fy: 0.4,
		radius: 1.35, 
		stop: 0 #efd, stop: 1 #888);
	min-width: 80px;
}
#navigationFrame QPushButton:hover {
	background: qradialgradient(cx: 0.3, cy: -0.4,
		fx: 0.3, fy: -0.4,
		radius: 1.35, stop: 0 #fff, stop: 1 #bbb);
}
#navigationFrame QPushButton:pressed {
	background: qradialgradient(cx: 0.4, cy: -0.1,
		fx: 0.4, fy: -0.1,
		radius: 1.35, stop: 0 #fff, stop: 1 #ddd);
}
#navigationFrame QPushButton:checked:hover {
	background: qradialgradient(cx: 0.3, cy: -0.4,
		fx: 0.3, fy: 0.4,
		radius: 1.35, 
		stop: 0 #efd, stop: 1 #bbb);
}



/* Sub Navigation Frame *************************************************/
/* largely inspired by http://thesmithfam.org/blog/2009/09/17/qt-stylesheets-button-bar-tutorial/ */
#subNavigationFrame {
	border: none;
	background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
		stop: 0 #a6a6a6, stop: 0.08 #7f7f7f,
		stop: 0.39999 #717171, stop: 0.4 #626262,
		stop: 0.9 #4c4c4c, stop: 1 #333333);
}
#subNavigationFrame QPushButton {
	color: #333;
	border: 2px solid #555;
	border-radius: 11px;
	padding: 5px;
	background: qradialgradient(cx: 0.3, cy: -0.4,
		fx: 0.3, fy: -0.4,
		radius: 1.35, stop: 0 #fff, stop: 1 #888);
	min-width: 80px;
}
#subNavigationFrame QPushButton:checked {
	color: #333;
	border: 2px solid #555;
	border-radius: 11px;
	padding: 5px;
	background: qradialgradient(cx: 0.3, cy: -0.4,
		fx: 0.3, fy: 0.4,
		radius: 1.35, 
		stop: 0 #fed, stop: 1 #888);
	min-width: 80px;
}
#subNavigationFrame QPushButton:hover {
	background: qradialgradient(cx: 0.3, cy: -0.4,
		fx: 0.3, fy: -0.4,
		radius: 1.35, stop: 0 #fff, stop: 1 #bbb);
}
#subNavigationFrame QPushButton:pressed {
	background: qradialgradient(cx: 0.4, cy: -0.1,
		fx: 0.4, fy: -0.1,
		radius: 1.35, stop: 0 #fff, stop: 1 #ddd);
}
#subNavigationFrame QPushButton:checked:hover {
	background: qradialgradient(cx: 0.3, cy: -0.4,
		fx: 0.3, fy: 0.4,
		radius: 1.35, 
		stop: 0 #fed, stop: 1 #bbb);
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>8</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="navigationFrame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>218</width>
        <height>0</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QCommandLinkButton" name="terrainCommandLink">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Terrain</string>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/toplevel-navigation/resources/dellipack_freeforcommercial_landscape.png</normaloff>:/toplevel-navigation/resources/dellipack_freeforcommercial_landscape.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>40</width>
           <height>48</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="description">
          <string>What we love the most.</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCommandLinkButton" name="forestCommandLink">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Forest &amp;&amp; Groundcover</string>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/toplevel-navigation/resources/greenpack_cc-attribution-noncommercial-sharealike_flora.png</normaloff>:/toplevel-navigation/resources/greenpack_cc-attribution-noncommercial-sharealike_flora.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>40</width>
           <height>48</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="description">
          <string>What we love the most.</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCommandLinkButton" name="sunskyCommandLink">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Sky &amp;&amp; Atmosphere</string>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/toplevel-navigation/skyatmosphere</normaloff>:/toplevel-navigation/skyatmosphere</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>40</width>
           <height>48</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="description">
          <string>... and time of day.</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCommandLinkButton" name="navigationCommandLink">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Navigation</string>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/toplevel-navigation/resources/oldschool_freefornoncommercial_navigation.png</normaloff>:/toplevel-navigation/resources/oldschool_freefornoncommercial_navigation.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>40</width>
           <height>48</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="description">
          <string>Find a spot!</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QCommandLinkButton" name="filmCommandLink">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Film</string>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/toplevel-navigation/resources/darkglass_gpl_film.png</normaloff>:/toplevel-navigation/resources/darkglass_gpl_film.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>40</width>
           <height>48</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="description">
          <string>Brighter? Darker?</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCommandLinkButton" name="renderingSetupCommandLink">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Rendering Setup</string>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/toplevel-navigation/resources/oxygen_gpl_screen.png</normaloff>:/toplevel-navigation/resources/oxygen_gpl_screen.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>40</width>
           <height>48</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="description">
          <string>Image-Size, Renderer, ...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCommandLinkButton" name="renderCommandLink">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Preview (F5)</string>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/toplevel-navigation/resources/sleekxp_cc-noncommercial_render.png</normaloff>:/toplevel-navigation/resources/sleekxp_cc-noncommercial_render.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>48</width>
           <height>48</height>
          </size>
         </property>
         <property name="shortcut">
          <string>F5</string>
         </property>
         <property name="description">
          <string>Action!</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string/>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="RenderWidget" name="renderWidget" native="true"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="mainFrame">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QStackedWidget" name="stackedWidget">
         <property name="currentIndex">
          <number>4</number>
         </property>
         <widget class="FilmSettingsWindow" name="filmSettings"/>
         <widget class="NavigationWindow" name="navigation"/>
         <widget class="TerrainWindow" name="terrain"/>
         <widget class="SunSkyWindow" name="sunSky"/>
         <widget class="TwinRenderSettingsWindow" name="renderingSetup"/>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>781</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSave"/>
    <addaction name="actionLoad"/>
   </widget>
   <widget class="QMenu" name="menu_Debug">
    <property name="title">
     <string>&amp;Debug</string>
    </property>
    <addaction name="action_Stylesheet"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menu_Debug"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSave">
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>&amp;Load</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="action_Stylesheet">
   <property name="text">
    <string>&amp;Stylesheet</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>RenderWidget</class>
   <extends>QWidget</extends>
   <header>renderwidget.hh</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FilmSettingsWindow</class>
   <extends>QWidget</extends>
   <header>filmsettingswindow.hh</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TerrainWindow</class>
   <extends>QWidget</extends>
   <header>terrainwindow.hh</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SunSkyWindow</class>
   <extends>QWidget</extends>
   <header>sunskywindow.hh</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>NavigationWindow</class>
   <extends>QWidget</extends>
   <header>navigationwindow.hh</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TwinRenderSettingsWindow</class>
   <extends>QWidget</extends>
   <header>twinrendersettingswindow.hh</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
